skrub/_total_std_norm.py

```diff
@@ -0,0 +1,100 @@
+import numpy as np
+
+
+def total_standard_deviation_norm(X):
```

Comment:
I think we should avoid talking about a "norm" as it isn't one, WDYT?

Reply:
I don't have a strong opinion, but on the surface it looks like a norm (e.g. the [RMS](https://pytorch.org/docs/stable/generated/torch.nn.RMSNorm.html#torch.nn.RMSNorm), with some differences: we apply the reduce on axis=0 instead of axis=1, and we sum instead of averaging). We can ask @GaelVaroquaux

Reply:
No strong opinion. But I like the term "scaler"